Aztec/Classes/TextKit/TextView.swift

```diff
@@ -674,17 +679,19 @@ open class TextView: UITextView {
         }
 
         let isBegginingOfListItem = storage.isStartOfNewLine(atLocation: range.location)
+        guard text == "\n" && beforeString == "\n" && afterString == "\n" && isBegginingOfListItem else {
```

Comment:
If you're going to keep working on these methods feel free to skip for now.  Let's just make that the final cut doesn't have magical strings.